runtime/compiler/control/JitDump.cpp

```diff
@@ -487,28 +487,28 @@ runJitdump(char *label, J9RASdumpContext *context, J9RASdumpAgent *agent)
       // dump event. This is typically used for debugging purposes so we process this option first if it exists.
       if (agent->dumpOptions != NULL)
          {
-         char* indexOfDot = strchr(agent->dumpOptions, '.');
-         char* indexOfParen = strchr(agent->dumpOptions, '(');
+         char *indexOfDot = strchr(agent->dumpOptions, '.');
+         char *indexOfParen = strchr(agent->dumpOptions, '(');
 
          if ((NULL != indexOfDot) && (NULL != indexOfParen))
             {
             TR::VMAccessCriticalSection findUserMethod(TR_J9VMBase::get(jitConfig, crashedThread));
 
-            U_8* className = reinterpret_cast<U_8*>(agent->dumpOptions);
-            U_16 classNameLength = indexOfDot - agent->dumpOptions;
-            U_8* methodName = reinterpret_cast<U_8*>(indexOfDot + 1);
-            U_16 methodNameLength = indexOfParen - indexOfDot - 1;
-            U_8* methodSig = reinterpret_cast<U_8*>(indexOfParen);
-            U_16 methodSigLength = strlen(indexOfParen);
+            char *className = agent->dumpOptions;
+            U_32 classNameLength = indexOfDot - agent->dumpOptions;
+            const char *methodName = indexOfDot + 1;
+            U_32 methodNameLength = indexOfParen - indexOfDot - 1;
+            const char *methodSig = indexOfParen;
+            U_32 methodSigLength = strlen(indexOfParen);
```

Comment:
`U_32` is not wide enough to hold a pointer difference - a cast is needed.

Reply:
Fixed in [force-push](https://github.com/eclipse/openj9/compare/70f787d25b95b995346cd4b3b411e35100b48f98..e6d4272c33428d33d17da82f1cddbbf549d82084).